recipe/meta.yaml

```diff
@@ -1,17 +1,20 @@
 {% set version = "0.6" %}
-{% set sha256 = "b4be2b3438971588dc306116658f475916755ad8fcc0618e79023846ce94f97b" %}
 
 package:
   name: python-graphviz
   version: {{ version }}
 
 source:
-  fn: python-graphviz-{{ version }}.zip
-  url: https://pypi.io/packages/source/g/graphviz/graphviz-{{ version }}.zip
-  sha256: {{ sha256 }}
+  fn: python-graphviz-{{ version }}.tar.gz
+  url: https://github.com/xflr6/graphviz/archive/{{ version }}.tar.gz
```

Comment:
> Do you mean changed checksums from the repo owner redefining the corresponding tag or file corruption from GitHub?

GitHub doesn't save the archive files. They are generated dynamically each time a user clicks one of the download links. Presumably they run `git archive` and do some other things. Despite doing some checks, they make no guarantee that the checksum will be stable between downloads. This often is due to things like timestamps being different or order files are added to the archive. So nothing nefarious necessarily, but annoying from a maintenance standpoint as we have to check and make sure it is ok.

I see. Could we just use CRLF in the patches or does that not work for some reason? In any event, I don't want to belabor this. Just curious to know more about what is happening. I'll take a look after merging.